### PR TITLE
Add e2e test: send nft 

### LINKED ISSUE #17184: sending an ERC721
Write an e2e test to cover the flow for sending an erc721
You can leverage the ganache seeder collectibles contract

### PR DESCRIPTION
**Pull request by chloeYue**
## Explanation

Fixes: #17184 

This PR adds e2e test for send NFT on UI.

## Pre-merge author checklist

 I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by digiwand — DISMISSED** [review]
LGTM!

File: test/e2e/nft/send-nft.spec.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        dapp: true,
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions,
+        smartContract,
+        title: this.test.title,
+      },
+      async ({ driver, _, contractRegistry }) => {
+        const contractAddress =
+          contractRegistry.getContractAddress(smartContract);
+        console.log(contractAddress);
```
Comment (darkwing) [inline test/e2e/nft/send-nft.spec.js:30]: Oops, we need to remove this `console.log`.

**Review by darkwing — CHANGES_REQUESTED** [review]
Just needs removal of a console statement.  Nice work!

File: test/e2e/nft/send-nft.spec.js (hunk 1 above)
Comment (chloeYue) [reply test/e2e/nft/send-nft.spec.js]: Ha, didn't see it. Removed, thanks!

**Review by NidhiKJha — APPROVED** [review]

**Review by PeterYinusa — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**